contracts/README.md

```diff
@@ -9,7 +9,7 @@ Interface that defines the events bridge contract implementations must emit. Als
 Interfaces that define the external functions for interacting with bridge contract implementations of each type. ERC20 and native token bridge interfaces vary from each other in that the native token bridge functions are `payable` and do not take an explicit amount parameter (it is implied by `msg.value`), while the ERC20 token bridge functions are not `payable` and require the explicit amount parameter. Otherwise, they include the same functions.
 
 ### `TeleporterTokenSource`
-An abstract implementation of `ITeleporterTokenBridge` for a bridge contract on the home chain with the asset to be bridged. Handles locking tokens to be sent to destination chains, as well as receiving bridge messages to either redeem tokens it holds as collateral (i.e unlock), or route them to another chain (i.e. "multi-hop"). In the case of a multi-hop transfer, the `TeleporterTokenSource` already has the collateral locked from when the tokens were originally bridged to the first destination chain, so it simply updates the accounting of the transferred balances to each respective destination.
+An abstract implementation of `ITeleporterTokenBridge` for a bridge contract on the home chain with the asset to be bridged. Handles locking tokens to be sent to destination chains, as well as receiving bridge messages to either redeem tokens it holds as collateral (i.e unlock), or route them to another chain (i.e. "multi-hop"). In the case of a multi-hop transfer, the `TeleporterTokenSource` already has the collateral locked from when the tokens were originally bridged to the first destination chain, so it simply updates the accounting of the transferred balances to each respective destination. Destination contracts must first be registered with a `TeleporterTokenSource` instance before the source contract will allow for sending tokens to them. This is to prevent tokens from being bridged to invalid destination addresses. Anyone is able to deploy and register destination contracts, and each destination must be evaluated independently for their security, and correctness.
```

Comment:
> and each destination must be evaluated independently for their security, and correctness.

Is this referring to the destination contract or subnet? The source contract doesn't do any "evaluation", so it's really up to the user to audit the destination chain + contract.

Reply:
Both I'd say. I was trying to get across the point that just because a destination is registered on a verified source contract does not mean it is trust worthy.

Reply:
Gotcha. IMO the wording is a bit vague. How about something like:

```suggestion
An abstract implementation of `ITeleporterTokenBridge` for a bridge contract on the home chain with the asset to be bridged. Handles locking tokens to be sent to destination chains, as well as receiving bridge messages to either redeem tokens it holds as collateral (i.e unlock), or route them to another chain (i.e. "multi-hop"). In the case of a multi-hop transfer, the `TeleporterTokenSource` already has the collateral locked from when the tokens were originally bridged to the first destination chain, so it simply updates the accounting of the transferred balances to each respective destination. Destination contracts must first be registered with a `TeleporterTokenSource` instance before the source contract will allow for sending tokens to them. This is to prevent tokens from being bridged to invalid destination addresses. Anyone is able to deploy and register destination contracts, which may have been modified from this repository. It is the responsibility of the users of the source contract to independently evaluate each destination for their security and correctness.
```

Reply:
SGTM!